model/LtiResultCustomFieldsService.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2018 (original work) Open Assessment Technologies SA;
+ *
+ * @author Ilya Yarkavets <ilya.yarkavets@1pt.com>
+ */
+
+namespace oat\ltiProctoring\model;
+
+use oat\taoDelivery\model\execution\DeliveryExecutionInterface;
+use oat\taoOutcomeUi\model\search\ResultCustomFieldsService;
+use oat\taoProctoring\model\deliveryLog\DeliveryLog;
+
+/**
+ * Class LtiResultCustomFieldsService
+ * @package oat\ltiProctoring\model
+ */
+class LtiResultCustomFieldsService extends ResultCustomFieldsService
```

Comment:
no tests found :)

Reply:
I just moved this method, not created or changed it

Reply:
Added a unit test